analyzer/emerald/emerald.go

```diff
@@ -218,7 +214,7 @@ func (m *Main) processRound(ctx context.Context, round uint64) error {
 		return err
 	}
 
-	opName := fmt.Sprintf("process_round_%s", emerald.String())
+	opName := fmt.Sprintf("process_round_rt%s", emerald.String())
```

Comment:
```suggestion
	opName := fmt.Sprintf("process_round_rt_%s", emerald.String())
```

Reply:
Warren suggested the non-underscore version [on slack](https://oasis-labs.slack.com/archives/C02UB6VEF0W/p1663697436482859), and we settled on it. I'm OK with the slight inconsistency, it makes it clear that the number is about the runtime.